jflac-codec/src/main/java/org/jflac/FLACDecoder.java

```diff
@@ -748,6 +767,8 @@ public Frame readNextFrame() throws IOException {
                     readFrame();
                     return frame;
                 } catch (FrameDecodeException e) {
+                	//e.printStackTrace();
```

Comment:
Fair enough, in that case perhaps remove the commented `printStackTrace()` and put a comment why it's ignored instead?